dev-python/PyQtWebEngine/PyQtWebEngine-5.12.1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+inherit python-r1 qmake-utils
+
+DESCRIPTION="Python bindings for Qt WebEngine framework"
+HOMEPAGE="https://www.riverbankcomputing.com/software/pyqtwebengine/intro"
+
+MY_P=${PN}_gpl-${PV/_pre/.dev}
+SRC_URI="https://www.riverbankcomputing.com/static/Downloads/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-python/PyQt5-5.12.1[${PYTHON_USEDEP},widgets]
+	>=dev-python/PyQt5-sip-4.19.14:=[${PYTHON_USEDEP}]
+	>=dev-qt/qtwebengine-5.12[widgets]"
+DEPEND="${RDEPEND}
+	>=dev-python/sip-4.19.14[${PYTHON_USEDEP}]"
+BDEPEND=""
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+	python_copy_sources
+}
+
+src_configure() {
+	configuration() {
+		local myconf=(
+			"${PYTHON}"
+			"${S}"/configure.py
+			$(usex debug '--debug --trace' '')
+			--destdir="$(python_get_sitedir)"/PyQt5
+			--pyqt-sipdir="$(python_get_scriptdir)"/../../../share/sip/PyQt5
```

Comment:
The default directory for this is `/usr/lib/python-exec/python3.6/../../../share/sip/PyQt5`, so it will fail if there is no python3.6 on the system. If I understand correctly in theory there may be such a configuration (e.g. `PYTHON_TARGETS="python2_7 python3_5"` or even `PYTHON_TARGETS="python2_7 python3_7"`).

Reply:
define "default directory"... is that what you see with `configure.py --help`? Unless upstream did things differently for this package, these paths are all derived from the version of the python interpreter used to run the script. Run `python3.7 configure.py --help` and you'll see a different path there.

Reply:
Oops, my bad. Thanks for the clarification.